src/app/tests/suites/certification/Test_TC_DRLK_2_13.yaml

```diff
@@ -0,0 +1,708 @@
+                  LockDataType: LockDataTypeEnum.AliroNonEvictableEndpointKey,
+                  DataOperationType: DataOperationTypeEnum.Add,
+                  OperationSource: OperationSourceEnum.Remote,
+                  UserIndex: 1,
+                  FabricIndex: fabric-idx,
+                  SourceNode: node-id,
+                  DataIndex: 2,
+              }
+
+    - label:
+          "Step 25: TH sends GetCredentialStatus Command with Credential as 8 1
+          (CredentialTypeEnum.AliroNonEvictableEndpointKey,CredentialIndex)"
+      PICS:
+          PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.C24.Rsp && DRLK.S.C25.Tx
+      command: "GetCredentialStatus"
+      arguments:
+          values:
+              - name: "Credential"
+                value: { CredentialType: 8, CredentialIndex: 1 }
+      response:
+          values:
+              - name: "CredentialExists"
+                value: true
+                constraints:
+                    type: boolean
+              - name: "UserIndex"
+                value: 1
+
+    - label:
+          "Step 26: TH sends ClearCredential Command to DUT to clear the
+          CredentialType of AliroEvictableEndpointKey Credential as 8 1
+          (CredentialTypeEnum.AliroNonEvictableEndpointKey,CredentialIndex)"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.C26.Rsp
+      command: "ClearCredential"
+      timedInteractionTimeoutMs: 1000
+      arguments:
+          values:
+              - name: "Credential"
+                value: { CredentialType: 8, CredentialIndex: 1 }
+
+    #NEED TO CHECK WITH VALUES
+    - label: "Step 27: TH reads the LockUserChange event list from DUT"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.F0d && DRLK.S.E04
+      command: "readEvent"
+      event: "LockUserChange"
+      response:
+          value:
+              {
+                  LockDataType: LockDataTypeEnum.AliroNonEvictableEndpointKey,
+                  DataOperationType: DataOperationTypeEnum.Clear,
+                  OperationSource: OperationSourceEnum.Remote,
+                  UserIndex: 1,
+                  FabricIndex: fabric-idx,
+                  SourceNode: node-id,
+                  DataIndex: 1,
+              }
+
+    - label:
+          "Step 28a: TH reads NumberOfAliroEndpointKeysSupported attribute and
+          saves the value as max_aliro_keys_supported"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.A0088
+      command: "readAttribute"
+      attribute: "NumberOfAliroEndpointKeysSupported"
+      response:
+          saveAs: max_aliro_keys_supported
+
+    - label:
+          "Step 28b: TH reads NumberOfCredentialsSupportedPerUser attribute and
+          saves the value as numberofcredentialsupportedperuser"
+      PICS: PICS_SKIP_SAMPLE_APP && DRLK.S.A001c
+      command: "readAttribute"
+      attribute: "NumberOfCredentialsSupportedPerUser"
+      response:
+          saveAs: numberofcredentialsupportedperuser
+
+    #############
+    #STEP 28c IF LOOP
```

Comment:
That part seems like it should probably a Python test, not YAML, no?

Reply:
We will be removing 2.13 to python so removed the file from this PR